servers/rendering_server.h

```diff
@@ -1642,6 +1648,7 @@ class RenderingServer : public Object {
 
 #ifndef DISABLE_DEPRECATED
 	void fix_surface_compatibility(SurfaceData &p_surface, const String &p_path = "");
+	void environment_set_fog_compat_84792(RID p_env, bool p_enable, const Color &p_light_color, float p_light_energy, float p_sun_scatter, float p_density, float p_height, float p_height_density, float p_aerial_perspective, float p_sky_affect);
```

Comment:
```suggestion
private:
	void _environment_set_fog_compat_84792(RID p_env, bool p_enable, const Color &p_light_color, float p_light_energy, float p_sun_scatter, float p_density, float p_height, float p_height_density, float p_aerial_perspective, float p_sky_affect);
	static void _bind_compatibility_methods();
```
You need to bind these, and they should be private (and prefixed with `_`)